src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -2120,6 +2120,35 @@ public struct Underline
             public float? Height { get; set; }
         }
 
+        /// <summary>
+        /// A struct to pass data of Strikethrough PropertyMap. <br />
+        /// </summary>
+        /// <remarks>
+        /// The Strikethrough struct is used as an argument to SetStrikethrough and GetStrikethrough methods. <br />
+        /// See <see cref="Tizen.NUI.BaseComponents.TextLabel.SetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextLabel.GetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextField.SetUnderline"/>, <see cref="Tizen.NUI.BaseComponents.TextField.GetStrikethrough"/>, <see cref="Tizen.NUI.BaseComponents.TextEditor.SetStrikethrough"/> and <see cref="Tizen.NUI.BaseComponents.TextEditor.GetStrikethrough"/>. <br />
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public struct Strikethrough
+        {
+            /// <summary>
+            /// Whether the strikethrough is enabled (the default value is false).
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public bool Enable { get; set; }
```

Comment:
```suggestion
            public bool Enabled { get; set; }
```

Reply:
Hi @myroot , We followed  the same properties naming of Underline. I do not know if this change could confuse user, what do you think ?

Reply:
If not yet published Underline API as officially, it also changed to `enabled` because it is commonly used naming

all apis use `enabled` naming instead `enable`

So, user will be confused with Underline API

Reply:
I checked the naming on NUIConstants.cs
only Underline and TextFit use Enable
and others use XXXEnabled

and fortunately, Underline and TextFit are not published so we can fit it

Reply:
@wonrst , please advise on this change-request because we have to apply it on DALi also.

If confirmed we will apply it for `Strikethrough` property only.
@SaraMohSamara , please update your patch when this confirmed, https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/258031
@abdullahghujah , please check your patch and update it  when this confirmed, https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/258031